### DISCUSSION TITLE
blogs/building-a-design-system-in-a-startup

### DISCUSSION THREAD

**Discussion by giscus** [highjoon/highjoon-dev#9] [Comments]
# blogs/building-a-design-system-in-a-startup

7개월 동안의 디자인 시스템 구축, 운영 및 실패 후기

https://highjoon-dev.vercel.app/blogs/building-a-design-system-in-a-startup

**Comment by qpwoei0123** [comment]
안녕하세요. 신규 프로젝트에 적용할 스타일 가이드 레퍼런스를 찾다가 이 디자인 시스템 회고까지 보게 되었네요. 정말 잘 읽었습니다.
저도 디자인 시스템을 고려하고 있었는데, 정말 섣불리 적용할 문제가 아니었네요. 머리가 띵한 느낌입니다... 좋은 글 감사합니다!
(참고로 블로그의 overflow-y 속성이 hidden으로 설정되어 있어서 개발자 도구로 직접 해제하고 스크롤했네요! 제 환경의 문제일 수도 있습니다.)

**Reply by highjoon** [reply]
안녕하세요. 도움이 되셨다니 다행입니다. 말씀해주신 현상은 수정했습니다. 감사합니다 🙂

**Comment by devkyoung2** [comment]
잘 읽고가요~~
